README.md

```diff
@@ -130,6 +131,26 @@ Experimental support for CGNS in the IOSS library is being added.  To use this c
      * `cd build`
      * `../../runconfigure.sh`
      * `make && make install`
+#### DataWarehouse
+The Data Warehouse is a collection of data management tools that Sandia is currently developing to improve how datasets migrate between memory and storage resources in a distributed system. While the software is currently only available internally to Sandia, there will be an open source release of the tools during Fall of 2017. The Data Warehouse extensions to SEACAS are intended to serve as a placeholder for the upcoming release and are not intended for use by external developers at this time.
+
+The repository [data-warehouse-release](https://gitlab.sandia.gov/nessie-dev/data-warehouse-release) is a superbuild for the Data Warehouse tools and was created to make the build process as easy as possible. It includes the files INSTALL.md and INSTALL_TPL.md which contain instructions for building Data Warehouse and it's TPLs: Boost, googletest, libfabric, and libhio (optional). These builds are straightforward so a "runconfigure.sh" script is left to the end-user. Note that it's possible to supply your own build of these tools. Following the SEACAS pattern for building TPLs:
+
+
+    mkdir SEACAS/TPL
```

Comment:
Should the be "cd SEACAS/TPL/DataWarehouse" instead of "mkdir SEACAS/TPL".  Not sure where the SEACAS/TPL subdirectory is meant to be created...